src/Kiota.Builder/KiotaBuilder.cs

```diff
@@ -935,7 +937,10 @@ private void CreateOperationMethods(OpenApiUrlTreeNode currentNode, OperationTyp
             {
                 DocumentationLink = operation.ExternalDocs?.Url,
                 DocumentationLabel = operation.ExternalDocs?.Description ?? string.Empty,
-                Description = (operation.Description ?? operation.Summary).CleanupDescription(),
+                Description = (operation.Description is string description && !string.IsNullOrEmpty(description) ?
+                                description :
+                                operation.Summary)
```

Comment:
can we safely assume that operation.Summary is non-null and not empty here?

Reply:
CleanUpDescription tolerates null or empty values. And since the summary is our last potential value here, it doesn't matter if we set the description to empty